docs/software/networking.mdx

```diff
@@ -15,3 +15,34 @@ robot, refer to the robot-specific tutorials, such as [this Husky tutorial](http
 Contact our Support team at support@clearpathrobotics.com if you have any questions.
 
 :::
+
+### SSH and SFTP
+
+Clearpath robots ship with an SSH server operating on port 22 for remote login.  This server also supports sftp for
+file-transfer to/from the robot if needed.  By default you can log in using the default username and password for the
+robot.
```

Comment:
Extra new line?

Reply:
Autopilot on my part; I always put two newlines before a section header, but I've removed the extras.